src/Tools/dotnet-monitor/Extensibility/IExtensionRepository.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Extensibility
+{
+    /// <summary>
+    /// Interface for service used to lookup an extension from a specific source
+    /// </summary>
+    internal interface IExtensionRepository
```

Comment:
Can we just keep the abstract class or interface? Or do we need both?

Reply:
Removed the `IExtensionRepository` and used `ExtensionRepository` directly. Also removed `IExtensionDiscoverer` and replaced with using `ExtensionDiscoverer` directly.